### PR TITLE
Adding docker-compose-test file for test service

### PR DESCRIPTION
**Pull request by KeerthanaR29**

### CODE REVIEW

File: docker-compose-test.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+services:
+  test:
+    container_name: react-app-test
+    build:
+      context: .
+      dockerfile: Dockerfile-test
+    volumes:
+      - './react-app:/usr/src/app'
+      - '/usr/src/app/node_modules'
```
Comment (SrinivasanTarget) [inline docker-compose-test.yaml:11]: `lint` `security` and `test` are missing isn't it?

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: docker-compose-test.yaml (hunk 1 above)
Comment (KeerthanaR29) [reply docker-compose-test.yaml:11]: Tried to execute it for test only,Will add lint and security

File: Dockerfile-test [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM node:16-alpine3.11
```
Comment (SrinivasanTarget) [inline Dockerfile-test:1]: Do we really need another dockerFile specific to test?

Comment (KeerthanaR29) [reply Dockerfile-test:1]: i tried to use dockerfile but it showed some error.Thats why created new Dockerfile for test

**Review by SrinivasanTarget — APPROVED** [review]

**Pull request merged**